docs/faults.rst

```diff
@@ -0,0 +1,32 @@
+View Faults (Supported in v2.4+)
+===============================================
+
+View App Faults
+---------------
+
+1. Traverse to top-right corner menu.
+
+2. Click :guilabel:`App Faults`.
+
+3. This displays a datagrid with the last 100 errors/warnings that occurred in the application’s background tasks and also displays resolution, if any.
+
+.. note::
+    - This specifically displays the faults in websocket and subscriptions for dynamic endpoint attach detach feature. 
+    
+    - To see older errors/warnings, please ssh to APIC server and check log files present in /data2/logs/F5Networks_F5ACIServiceCenter/faults/app.log
```

Comment:
...errors/warnings, SSH to the APIC server and check the log files in /data2....